unittest/buildtool/bom_command_test.py

```diff
@@ -275,11 +274,11 @@ def test_build(self):
         PATCH_VERSION_NUMBER + '-SourceInfoBuildNumber')
     golden_bom['services']['monitoring-third-party']['version'] = (
         PATCH_VERSION_NUMBER + '-SourceInfoBuildNumber')
-    
+
     golden_bom['artifactSources'] = {
       'debianRepository': 'https://dl.bintray.com/%s/%s' % (
           options.bintray_org, options.bintray_debian_repository),
-      'dockerRegistry': options.docker_registry,
+      'dockerRegistry': options.artifact_registry,
```

Comment:
I think this confirms for me that sticking with `docker_registry` as the variable name is the way to go.

Reply:
Please see above. The test was updated to reflect current code. It would be good to rename/refactor when can.